boto3/docs/service.py

```diff
@@ -22,9 +24,12 @@
 
 
 class ServiceDocumenter(object):
-    def __init__(self, service_name, session):
+    def __init__(self, service_name, session, examples_file=None):
```

Comment:
Another idea is that we can make the examples directory a ``EXAMPLES_DIRECTORY`` in the class that you can override when you instantiate the class and move the ``_get_examples_file()`` to when the ``document_service()`` call is made.  As this argument seems to only be needed for testing.